backends/qualcomm/serialization/qc_compiler_spec.fbs

```diff
@@ -35,7 +36,9 @@ enum QcomChipset: int {
   SM8475 = 42,
   SM8550 = 43,
   SSG2115P = 46,
+  SM8635 = 68,
```

Comment:
Unrelated - we may need to figure out how to make it non BC breaking. I remember extending in flatbuffers won't be BC breacking, but it'll a bit annoying if it's not ordered...